config/nat.py

```diff
@@ -1,8 +1,14 @@
 import ipaddress
 
 import click
+import jsonpatch
+import jsonpointer
+from jsonpatch import JsonPatchConflict
+from jsonpointer import JsonPointerException
 from swsscommon.swsscommon import SonicV2Connector, ConfigDBConnector
+from .validated_config_db_connector import ValidatedConfigDBConnector
 
+ADHOC_VALIDATION = True
```

Comment:
Every py file update config has defined ADHOC_VALIDATION, is there any conflict?

Reply:
There is no conflict, as the variable scope is limited to the file